SU-HSDO/stanford_mrc
# MRCD8-270 masthead styles

# READY FOR REVIEW
- Waiting on mobile menu to be completed before finalizing responsiveness

# Summary
- Floats the search form to the right
- This PR works in partnership with another (noted below) in stanford_basic

# Needed By (Date)
- End of sprint (3/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Pull branch of same name in stanford_basic
3. Ensure that site title lockup is spanning 2/3 and search is spanning 1/3 and that they are inline with one another

# Affected Projects or Products
- MRC
- stanford_basic

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-270
- https://github.com/SU-SWS/stanford_basic/pull/56

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
